hack/manifests/testing/ingress.yaml

```diff
@@ -3,14 +3,23 @@ apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
   name: parodos-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /$2
 spec:
   rules:
   - http:
       paths:
       - pathType: Prefix
-        path: /
+        path: /workflow-service(/|$)(.*)
         backend:
           service:
             name: workflow-service
             port:
               number: 8080
+      - pathType: Prefix
+        path: /notification-service(/|$)(.*)
```

Comment:
same for this.

Reply:
You're right!

WDYT? 
https://github.com/gciavarrini/backstage-parodos/tree/add-testing-services-paths 
See commit https://github.com/parodos-dev/backstage-parodos/compare/main...gciavarrini:backstage-parodos:add-testing-services-paths

( I haven't opened the PR yet, I'd like to have an early feedback)

Reply:
seems good to me, but requires verification and feedback from parodos-backstage maintainers.